### PR TITLE
adding code for lab8(java lab2)

### PR DESCRIPTION
**Pull request by ArSiu**

### CODE REVIEW

File: src/main/java/org/arsiu/Main.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        manager.tech.add(new InkjetPrinter("INKJ2000", "3202", COMPANIES.PANASONIC, 2019,
+                299.9f, 220, 60, 100, 100, 10));
+        manager.tech.add(new LaserPrinter("LP-2040", "4550", COMPANIES.PANASONIC, 2020,
+                200.0f, 220, 160, 150, 150,true));
+        manager.tech.add(new CuttingPlotter("CT25", "3542", COMPANIES.BOSCH, 2009, 50,
+                50, 100, "STL 090"));
+        manager.tech.add(new Shredder("RoyTom", "model-Y", COMPANIES.PHILIPS, 2007, 30,
+                220, 200, 12, 24));
+
+        //manager.sortByName(true).forEach(System.out::println);
```
Comment (zveres-ss) [inline src/main/java/org/arsiu/Main.java]: either use this code or remove it from codebase

File: src/main/java/org/arsiu/file/FileType.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package org.arsiu.file;
+
+public class FileType{
+    public enum FILETYPE{
```
Comment (zveres-ss) [inline src/main/java/org/arsiu/file/FileType.java]: Enum name vilates convention. Do not hide enum inside the class
Class FileType is useless

File: src/main/java/org/arsiu/technique/CuttingPlotter.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import lombok.Setter;
+import org.arsiu.technique.Companies.COMPANIES;
+
+@Setter
+@Getter
+@NoArgsConstructor
+public class CuttingPlotter extends Technique {
+    public String nameOfBlade;
+
+    public CuttingPlotter(String name, String model, COMPANIES manufactoryCompany, int yearOfManufactory, float price,
```
Comment (zveres-ss) [inline src/main/java/org/arsiu/technique/CuttingPlotter.java]: COMPANIES manufactoryCompany -> CompanyType manufactoryCompany or change manufactoryCompany type to String

File: src/main/java/org/arsiu/technique/CuttingPlotter.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return objToString() +
+                "|\n|Name of blade: " + nameOfBlade +
+                "|\n________________________\n";
+    }
+
+    public void cut() {
```
Comment (zveres-ss) [inline src/main/java/org/arsiu/technique/CuttingPlotter.java:32]: what is the purpose of this method?

File: src/main/java/org/arsiu/technique/Shredder.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+public class Shredder extends Technique {
+    public int amountOfBlades;
```
Comment (zveres-ss) [inline src/main/java/org/arsiu/technique/Shredder.java]: never use public modifier for fields

**Review by zveres-ss — CHANGES_REQUESTED** [review]

File: src/main/java/org/arsiu/Main.java (hunk 1 above)
Comment (ArSiu) [reply src/main/java/org/arsiu/Main.java]: fixed

File: src/main/java/org/arsiu/file/FileType.java (hunk 2 above)
Comment (ArSiu) [reply src/main/java/org/arsiu/file/FileType.java]: yeap

File: src/main/java/org/arsiu/technique/CuttingPlotter.java (hunk 3 above)
Comment (ArSiu) [reply src/main/java/org/arsiu/technique/CuttingPlotter.java]: ugu

File: src/main/java/org/arsiu/technique/CuttingPlotter.java (hunk 4 above)
Comment (ArSiu) [reply src/main/java/org/arsiu/technique/CuttingPlotter.java:32]: Write cut. It is something like simulating a real working plotter. Yea maybe stupid but works fine.

File: src/main/java/org/arsiu/technique/Shredder.java (hunk 5 above)
Comment (ArSiu) [reply src/main/java/org/arsiu/technique/Shredder.java]: Aga changed to private.

**Pull request merged**